foodsaving/subscriptions/serializers.py

```diff
@@ -0,0 +1,33 @@
+from rest_framework import serializers
+from rest_framework.fields import SerializerMethodField, CurrentUserDefault
+from rest_framework.validators import UniqueTogetherValidator
+
+from foodsaving.subscriptions.models import PushSubscription, PushSubscriptionPlatform
+from foodsaving.users.serializers import UserIdSerializer
+
+
+class PushSubscriptionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = PushSubscription
+        fields = [
+            'id',
+            'user',
+            'token',
+            'platform'
+        ]
+        validators = [
+            UniqueTogetherValidator(
+                queryset=PushSubscription.objects.all(),
+                fields=('user', 'token')
+            )
+        ]
+
+    user = UserIdSerializer(
```

Comment:
Adding this was the only way I could get it to respect the `UniqueTogetherValidator`. I don't actually want the user info in the serializer, and adding/using `UserIdSerializer` was just to minimize the extra stupid data.

I suspect there is a better way to solve it.

Reply:
Seems a good workaround, maybe add a short comment in the code?